src/main/java/ch/cern/eam/wshub/core/services/workorders/entities/WorkOrderActivityCheckListSignature.java

```diff
@@ -0,0 +1,125 @@
+package ch.cern.eam.wshub.core.services.workorders.entities;
+
+import ch.cern.eam.wshub.core.annotations.InforField;
+import net.datastream.schemas.mp_fields.ACTIVITYCODE;
+import net.datastream.schemas.mp_fields.NOUN_Type;
+import net.datastream.schemas.mp_fields.VERB_Type;
+
+import java.io.Serializable;
+import java.math.BigInteger;
+
+public class WorkOrderActivityCheckListSignature implements Serializable{
```

Comment:
Is this class transformed from an Infor class? Why does it have the InforField annotations?

Reply:
Used for transformation in reviewWorkOrderActivityCheckList and performWorkOrderActivityChecklist.